js/interface.js

```diff
@@ -342,6 +342,20 @@ function loadTrashFolder() {
   updateSearchTypeOptions($element.data('type'));
 }
 
+function restoreAction(type, id) {
+  var url = 'v1/media/' + type + '/' + id + '/restore';
+
+  return Fliplet.API.request({
+    url: url,
+    method: 'POST',
+  }).then(function() {
+    checkboxStatus();
+    completedItems++;
```

Comment:
Why completed? Can you clarify the naming behind this variable and how it's used?

Reply:
CompleteItems - this variable about how many items have been restored, it is needed in order to understand whether this is a multiple deletion or one, and to understand whether the restoration is successful

Reply:
Then call it "restoredItems"?

Reply:
your right, changed